<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>135</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>System</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Меню</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>System</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Количество значащих цифр в двоичной записи</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сумма цифр или разрядов числа в n-ой системе</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Найти количество цифр или первую цифру числа в n-ой системе</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Поиск частоты цифры числа в n-ой системе</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>N-основание системы числа</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>System</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выберите тип задания для решения</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
